tests/unit/identities/test_trait_schema.py

```diff
@@ -28,13 +31,26 @@ def test_trait_schema_load(trait_key, trait_value):
     assert trait_model.trait_value == trait_value
 
 
+def test_trait_schema_load_converts_decimal_to_float():
+    # Given
+    trait_value = 11.9
+    trait_schema = TraitSchema()
+
+    # When
+    trait_model = trait_schema.load(
+        {"trait_key": "key", "trait_value": decimal.Decimal(str(trait_value))}
+    )
+
+    # Then
+    assert trait_model.trait_value == trait_value
```

Comment:
I think it would be good to have an assertion here on `isinstance(trait_model.trait_value, float)` since that's what the test is described as. I appreciate this is essentially the same thing but I think it's more obvious on a quick glance if we add that in.